.github/workflows/build-on-ubuntu.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
       - uses: actions/setup-java@v2
         with:
-          java-version: 11
+          java-version: 8
```

Comment:
Please revert here and in other `.yml` files.

BTW, I have no idea how the builds pass with this.

Reply:
That should be the Gradle's JVM toolchain, which completely ignores what's Java version is provided by the OS...